src/hooks/useConfig.ts

```diff
@@ -0,0 +1,80 @@
+import Path from "path";
+import { Verbosity } from "../types.ts";
+import { set_tmp } from "path"
+
+export interface Env {
+  CI?: string
+  CLICOLOR?: string
+  CLICOLOR_FORCE?: string
+  DEBUG?: string
+  GITHUB_ACTIONS?: string
+  GITHUB_TOKEN?: string
+  NO_COLOR?: string
+  PATH?: string
+  RUNNER_DEBUG?: string
+  SHELL?: string
+  SRCROOT?: string
+  TEA_DIR?: string
+  TEA_FILES?: string
+  TEA_FORK_BOMB_PROTECTOR?: string
+  TEA_PANTRY_PATH?: string
+  TEA_PKGS?: string
+  TEA_PREFIX?: string
+  TEA_REWIND?: string
+  VERBOSE?: string
+  VERSION?: string
+}
+
+export interface Config {
+  isCI: boolean
+
+  execPath: Path
+  loggerGlobalPrefix?: string
+  teaPrefix: Path
+
+  verbosity: Verbosity
+  dryrun: boolean
+  keepGoing: boolean
+
+  verbose: boolean
+  debug: boolean
+  silent: boolean
+
+  env: Env
```

Comment:
I wonder if this (or `Env`) needs a `raw: string[]` value for future sanity. If we don't need it now, then maybe we're fine.

Reply:
raw would need to return `{key: string, value: string}[]` right?

Reply:
hm. yes. or possibly `Record<string, string>[]`.